tests/core/full_node/test_mempool.py

```diff
@@ -565,7 +538,12 @@ async def condition_tester(
             farmer_reward_puzzle_hash=reward_ph,
             pool_reward_puzzle_hash=reward_ph,
         )
-        peer = await connect_and_get_peer(server_1, server_2, bt.config["self_hostname"])
+        _, dummy_node_id = await add_dummy_connection(server_1, bt.config["self_hostname"], 100)
+        dummy_peer = None
+        for node_id, wsc in server_1.all_connections.items():
+            if node_id == dummy_node_id:
+                dummy_peer = wsc
+                break
```

Comment:
it looks like `connect_and_get_peer()` was useful. We do this at least twice. Why remove it?

Reply:
connect and get peer is for two nodes to connect to each other. But we don't actually need 2 nodes for this test. Creating unnecessary node processes slows things down due to the setup time. Here a dummy peer is sufficient.

Reply:
I see. `connect_and_get_peer()` did more than this block

Reply:
yeah but the main problem is the fixture which used to get 2 nodes, now it only gets one.